src/kernel.ts

```diff
@@ -900,9 +887,9 @@ class KernelFutureHandler extends Disposable implements IKernelFuture {
         }
       }
     } else if (msg.channel === 'shell') {
-      var reply = this._output;
+      var reply = this._reply;
```

Comment:
Note that the test suite is only about half complete, I had stopped working on it to figure out packaging.